internal/binutils/binutils_test.go

```diff
@@ -401,3 +414,105 @@ func TestOpenMalformedMachO(t *testing.T) {
 		t.Errorf("Open: got %v, want error containing 'Mach-O'", err)
 	}
 }
+
+func TestObjdumpVersionChecks(t *testing.T) {
+	// Test that the objdump version strings are parsed properly.
+
+	var version string
+	if runtime.GOOS == "darwin" {
```

Comment:
Consider making this test a table-driven test, which is common in Go? https://github.com/golang/go/wiki/TableDrivenTests

Reply:
Done.